ups/sims_seeingModel.table

```diff
@@ -0,0 +1,14 @@
+setupRequired(utils >= 4.6.0.0)
+# These packages are optional for running sims_seeingModel, but required for tests.
+# Note that 'eups distrib' installs and sets up setupOptional packages.
+# TimeHandler lives in sims_utils
+setupOptional(sims_utils)
+# Throughputs and sims_photUtils are required to automatically set the effwavelens
+# (and this is tested in the unit tests). Defaults will be used if these packages
+# are not installed.
+setupOptional(throughputs)
+setupOptional(sims_photUtils)
```

Comment:
I would suggest to remove the optional dependency here as well.

Reply:
I see... But, isn't `sims_seeingModel` supposed to be system agnostic? I imagine that, if an external package uses `sims_seeingModel` and needs to check the throughputs, than it should be done on the external package.

Reply:
It's actually not completely system agnostic. The seeingModel is built on simulations that Bo Xin ran for LSST. I think that at the very least, the relationship between FWHMeff and FWHMgeom would be different for different systems. 
That said, you could always use this for different systems (or more likely, LSST with different filters) by specifying the wavelengths yourself.